finance-api/user-guides/fetch-purchase-information-for-transactions.md

```diff
@@ -0,0 +1,168 @@
+Fetch purchase information for transactions
+===
+Using the Finance API and the Purchase API, you can fetch purchase information for transactions. The `originatingTransactionUuid` of a transaction in the Finance API corresponds to the `uuid` of `payments` in the Purchase API.  
+> **Note:** Currently, fetching purchase information is available only for transactions that are made with cards and alternative payment methods (APMs) like PayPal QRC through Zettle.
+
+The Finance API records transactions for payment types that are supported by the Purchase API except `IZETTLE_CASH`, `SWISH`, `VIPPS`, `MOBILE_PAY`, `GIFTCARD`, `STORE_CREDIT`, and `IZETTLE_INVOICE`. 
+> **Note:**  If `IZETTLE_INVOICE` is paid with a card, the purchase will be recorded as `CARD_PAYMENT` in the Finance API.
+
+The following table shows the payment methods, payment types that are supported by the Purchase API, and the corresponding transaction types that are supported by the Finance API. 
+
+|Payment methods |Payment types by the Purchase API |Corresponding transaction type by the Finance API |
+|:---|:--- |:--- 
+|Card payment |`IZETTLE_CARD` |`CARD_PAYMENT`, `CARD_PAYMENT_FEE` |
+|Card payment online |`IZETTLE_CARD_ONLINE` |`CARD_PAYMENT`, `CARD_PAYMENT_FEE` | 
+|Card refund |`IZETTLE_CARD` |`CARD_REFUND`, `CARD_PAYMENT_FEE_REFUND` |
+|APM |`PayPal QRC`, `KLARNA` |`PAYMENT`, `PAYMENT_FEE` |  
+
+* [Prerequisites](#prerequisites)
+* [Step 1: Find the transaction UUID with the Finance API](#step-1-find-the-transaction-uuid-with-the-finance-api)
+* [Step 2: Fetch purchase information for transactions with the Purchase API](#step-2-fetch-purchase-information-for-transactions-with-the-purchase-api)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorisation is set up with the following OAuth scope using [OAuth2 API](../../authorization.adoc):
+    * `READ:FINANCE`
+    * `READ:PURCHASE`
+
+## Step 1: Find the transaction UUID with the Finance API
+Currently, fetching purchase information is available only for transactions that are made with cards, PayPal QRC, and Klarna. 
```

Comment:
```suggestion
Currently, fetching purchase information is available only for transactions made with cards, PayPal QRC, and Klarna. 
```

Reply:
Fixed the sentence to be aligned with the note